To: vim_dev@googlegroups.com
Subject: Patch 7.3.475
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.475
Problem:    In a terminal with few colors the omnicomplete menu may be hard to
	    see when using the default colors.
Solution:   Use more explicit colors. (suggested by Alex Henrie)
Files:	    src/syntax.c


*** ../vim-7.3.474/src/syntax.c	2012-01-10 22:26:12.000000000 +0100
--- src/syntax.c	2012-03-16 20:14:22.000000000 +0100
***************
*** 6516,6523 ****
  	     "DiffText term=reverse cterm=bold ctermbg=Red gui=bold guibg=Red"),
  #endif
  #ifdef FEAT_INS_EXPAND
- 	CENT("PmenuThumb cterm=reverse",
- 	     "PmenuThumb cterm=reverse gui=reverse"),
  	CENT("PmenuSbar ctermbg=Grey",
  	     "PmenuSbar ctermbg=Grey guibg=Grey"),
  #endif
--- 6516,6521 ----
***************
*** 6557,6566 ****
  	     "SpellLocal term=underline ctermbg=Cyan guisp=DarkCyan gui=undercurl"),
  #endif
  #ifdef FEAT_INS_EXPAND
! 	CENT("Pmenu ctermbg=LightMagenta",
! 	     "Pmenu ctermbg=LightMagenta guibg=LightMagenta"),
! 	CENT("PmenuSel ctermbg=LightGrey",
! 	     "PmenuSel ctermbg=LightGrey guibg=Grey"),
  #endif
  	CENT("SpecialKey term=bold ctermfg=DarkBlue",
  	     "SpecialKey term=bold ctermfg=DarkBlue guifg=Blue"),
--- 6555,6566 ----
  	     "SpellLocal term=underline ctermbg=Cyan guisp=DarkCyan gui=undercurl"),
  #endif
  #ifdef FEAT_INS_EXPAND
! 	CENT("PmenuThumb ctermbg=Black",
! 	     "PmenuThumb ctermbg=Black guibg=Black"),
! 	CENT("Pmenu ctermbg=LightMagenta ctermfg=Black",
! 	     "Pmenu ctermbg=LightMagenta ctermfg=Black guibg=LightMagenta"),
! 	CENT("PmenuSel ctermbg=LightGrey ctermfg=Black",
! 	     "PmenuSel ctermbg=LightGrey ctermfg=Black guibg=Grey"),
  #endif
  	CENT("SpecialKey term=bold ctermfg=DarkBlue",
  	     "SpecialKey term=bold ctermfg=DarkBlue guifg=Blue"),
***************
*** 6645,6654 ****
  	     "SpellLocal term=underline ctermbg=Cyan guisp=Cyan gui=undercurl"),
  #endif
  #ifdef FEAT_INS_EXPAND
! 	CENT("Pmenu ctermbg=Magenta",
! 	     "Pmenu ctermbg=Magenta guibg=Magenta"),
! 	CENT("PmenuSel ctermbg=DarkGrey",
! 	     "PmenuSel ctermbg=DarkGrey guibg=DarkGrey"),
  #endif
  	CENT("Title term=bold ctermfg=LightMagenta",
  	     "Title term=bold ctermfg=LightMagenta gui=bold guifg=Magenta"),
--- 6645,6656 ----
  	     "SpellLocal term=underline ctermbg=Cyan guisp=Cyan gui=undercurl"),
  #endif
  #ifdef FEAT_INS_EXPAND
! 	CENT("PmenuThumb ctermbg=White",
! 	     "PmenuThumb ctermbg=White guibg=White"),
! 	CENT("Pmenu ctermbg=Magenta ctermfg=Black",
! 	     "Pmenu ctermbg=Magenta ctermfg=Black guibg=Magenta"),
! 	CENT("PmenuSel ctermbg=DarkGrey ctermfg=Black",
! 	     "PmenuSel ctermbg=DarkGrey ctermfg=Black guibg=DarkGrey"),
  #endif
  	CENT("Title term=bold ctermfg=LightMagenta",
  	     "Title term=bold ctermfg=LightMagenta gui=bold guifg=Magenta"),
*** ../vim-7.3.474/src/version.c	2012-03-16 19:34:43.000000000 +0100
--- src/version.c	2012-03-16 20:05:35.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     475,
  /**/

-- 
Spam seems to be something useful to novices.  Later you realize that
it's a bunch of indigestable junk that only clogs your system.
Applies to both the food and the e-mail!

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
